package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "csharp",
   "publisher": "ms-vscode",
-  "version": "1.3.0",
+  "version": "1.3.0-beta1",
```

Comment:
Nope! :smile: See my email on releases that I sent last week.

Once I merge this, I'll put up a pre-release out of master for folks to test.